script.js

```diff
@@ -1,4 +1,184 @@
+/*Base
+Requirements
+There are 2 players and players take turns.
+When a player clicks Submit, the game rolls 2 dice and shows the dice rolls, for example 3 and 6.
+The player picks the order of the dice they want. For example, if they wanted the number 63, they would specify that the 2nd dice goes first. You can choose how the player specifies dice order.
+After both players have rolled and chosen dice order, the player with the higher combined number wins.*/
+
+// -1: Introduction and game mode selection
+// 0 : Before player 1 roll dice
+// 1 : After player 1 roll dice
+// 2 : Before player 2 roll dice
+// 3 : After player 2 roll dice
+var turnOrOrder = -1;
+
+// 1: Normal mode
+// 2: Reversed Mode
+var gameMode = 0;
+
+// 0 : No action needed
+// 1 : Action needed
+var whichDiceOrder = 0;
+
+// to track overall results
+var pOneStats = 0;
+var pTwoStats = 0;
+var leaderboard = "";
+var myOutputValue;
+
+var playerThrow = [0, 0];
+var pOneResult = 0;
+var pTwoResult = 0;
+
+// Dice roll function to get random dice number
+var diceRollResult = function () {
+  var rollResult = Math.floor(Math.random() * 6) + 1;
+  return rollResult;
+};
+
+// Evaluate results
+var evaluateResults = function (input) {
```

Comment:
`evaluateResults` is a huge function so do consider how you can refactor it. Your gameplay logic and code can be broken into three distinct parts: 1. players roll, 2. players select order, 3. compare score. This provides a very nice starting point on how we can split `evaluateResults` further into separate functions.